### PR TITLE
git-subrepo: install shell completion

### PR DESCRIPTION
**Pull request by cdlm**

### CODE REVIEW

File: Formula/git-subrepo.rb [hunk 1]
```diff
@@ -16,6 +16,15 @@ def install
     libexec.mkpath
     system "make", "PREFIX=#{prefix}", "INSTALL_LIB=#{libexec}", "install"
     bin.install_symlink libexec/"git-subrepo"
+
```
Comment (fxcoudert) [inline Formula/git-subrepo.rb]: Can you bump revision? (add a `revision 1` line before `head`)

**Review by fxcoudert — CHANGES_REQUESTED** [review]

File: Formula/git-subrepo.rb (hunk 1 above)
Comment (cdlm) [inline Formula/git-subrepo.rb]: Sure, done. Should I squash both commits?

Comment (fxcoudert) [inline Formula/git-subrepo.rb]: Yes please

**Review by fxcoudert — APPROVED** [review]

**Comment by fxcoudert** [comment]
Thanks @cdlm for your contribution!

**Pull request closed without merging**